docs/oef-ledger.md

```diff
@@ -13,7 +13,7 @@ We will soon make a public OEF network available.
 
 
 Ledgers enable the AEAs to complete a transaction and transfer funds to each other. Another way the AEA framework uses 
-the ledgers are to interact with smart contracts.
+the ledgers are to interact with smart contracts. Whilst a ledger can, in principle also be used to store structured data - for instance training data in a machine learning model - in most use cases the resulting costs and privacy implications do not make this a relevant use of the ledger. Instead, usually only references of the structured data - often in the form of hashes - are stored on the ledger and the actual data is stored off-chain.
```

Comment:
Some minor things:

- Whilst a ledger can, in principle*, *also
- for instance*, *training data
- Instead, usually only references *to* the structured data